common/linear_actuator/linear_actuator.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component version="1.2" xmlns="http://schemas.circuit-diagram.org/circuitDiagramDocument/2012/component/xml">
+  <declaration>
+
+    <!-- Describe the component -->
+    <meta name="name" value="Linear Actuator" />
+    <meta name="version" value="1.0" />
+    <meta name="minsize" value="100" />
+    <meta name="author" value="Michael Salvini" />
+    <meta name="guid" value="92616277-5725-436f-b860-02e3bba18f40" />
+    
+    <flags>
+      <option>horizontalonly</option>
+    </flags>
+
+  </declaration>
+  <connections>
+
+    <group conditions="horizontal">
```

Comment:
The component can only be placed horizontally, so the conditions attribute is not necessary here:

```suggestion
    <group>
```